### DISCUSSION TITLE
Project dead?

### DISCUSSION THREAD

**Discussion by BenWallmeyer** [thedevdojo/voyager#5781] [General]
I set up a lot of voyager projects, but now i see no bugfixes, no laravel 10 support and the last commit is long time ago.

Now the question to the-control-group.

Is tcg voyager dead?

**Comment by rodrigolopespt** [comment]
Move to Laravel Filament

**Reply by gcorp-llc** [reply]
Filament is very slow even though it uses Livewire 3 and many of its plugins are not compatible with the new version.
Voyager's tools cannot be compared to Filament
But there is no choice, the best choice after Voyager is to use filament
